index.rkt

```diff
@@ -360,6 +360,10 @@ compiler, and @italic{that} syntax is evaluated:
 (values "i" "am" "backwards")
 ]
 
+@margin-note{The first argument of @racket[syntax->datum] contains the lexical
+context information that we want to associate with the @racket[syntax]
+outputted by the transformer.}
+
 @; ----------------------------------------------------------------------------
```

Comment:
I've put it in a margin note to avoid breaking the flow of the explanation that transitions nicely with the examples shown.